src/modules/test/README.md

```diff
@@ -1,11 +1,17 @@
-# modulestest
-The `modulestest` application is OSConfig’s module testing framework, it allows module authors to write test recipes which in turn, drive testing of the module functionality and conformity (against the published MIM) of the Management Module.
+# ModulesTest
+The `modulestest` utility is OSConfig’s module testing framework, it allows module authors to write Test Recipes which in turn, drive testing of the module functionality and conformity (against the published MIM) of the Management Module.
 
 # Prerequisites
-Although `modulestest` does not require any runtime dependencies, generating the test recipe configuration (testplate.json) does require the `jq` application to be installed when building OSConfig. When installed a `testplate.json` is placed in the `modulestest` directory which includes the test recipe configurations for every module which includes a [MIM](https://github.com/Azure/azure-osconfig/tree/main/src/modules/mim) and [test recipe](https://github.com/Azure/azure-osconfig/tree/main/src/modules/test/recipes).
+Although `modulestest` does not require any runtime dependencies, generating the test recipe configuration (testplate.json) does require the `jq` application to be installed when building OSConfig.

```

Comment:
>`jq` [](http://example.com/codeflow?start=141&length=4)

What means jq? Where does this comes from? Can you put the full name or get a hyperlink to that GitHub repo if there is one or the tool?